sass/components/_content_organisation.scss

```diff
@@ -0,0 +1,10 @@
+/**
+ * @file
+ * This file is used to style the Organisation content.
+ */
+
+.figure-transparent {
+  figure.ecl-media-container {
+    background-color: transparent !important;
+  }
+}
```

Comment:
See above, let's try to remove this if possilbe

Reply:
removed